AWESOME.md

```diff
@@ -8,8 +8,25 @@ We will be happy to add your success story using AutoGluon to this list.
+| :tv: [AutoGluon: AutoML for Tabular, Multimodal, and Time Series Data](https://youtu.be/Lwu15m5mmbs?si=jSaFJDqkTU27C0fa)                                           | Tutorial | PyData Berlin                                                                                                                      | 2023/06/20 | 
+| :tv: [Solving Complex ML Problems in a few Lines of Code with AutoGluon](https://www.youtube.com/watch?v=J1UQUCPB88I)                                              | Tutorial | PyData Seattle                                                                                                                     | 2023/06/20 | 
+| [AutoGluon: Empowering (Multimodal) AutoML for the Next 10 Million Users](https://autogluon.github.io/neurips2022-autogluon-workshop/)                             | Tutorial | [NeurIPS 2022](https://nips.cc/Expo/Conferences/2022/workshop/63089)                                                               | 2022/11/28 |
+| :tv: [The AutoML Revolution](https://www.youtube.com/watch?v=VAAITEds-28)                                                                                          | Tutorial | [Fall AutoML School 2022](https://sites.google.com/view/automl-fall-school-2022)                                                   | 2022/10/18 |
+| [Multimodal AutoML for Image, Text and Tabular Data](https://github.com/Innixma/kdd-2022-multimodal-automl-tutorial)                                               | Tutorial | [KDD 2022](https://kdd.org/kdd2022/lectureTutorial.html)                                                                           | 2022/08/14 |
+| :tv: [Automating Machine Learning for the Rest of Us](https://www.youtube.com/watch?v=x-EEDBSl3OM)                                                                 | Keynote  | [AutoML Conf 2022](https://2022.automl.cc/index.html)                                                                              | 2022/07/25 |
+| :tv: [AutoGluon: re:MARS 2022 Keynote](https://www.youtube.com/watch?v=_wd0IJBTwbY&t=1277s)                                                                        | Keynote  | [Amazon re:MARS 2022](https://icml.cc/virtual/2020/workshop/5725)                                                                  | 2022/06/24 |
+| [Advancing the State of the Art in AutoML](https://developer.nvidia.com/blog/advancing-the-state-of-the-art-in-automl-now-10x-faster-with-nvidia-gpus-and-rapids/) | Talk     | [GTC 2021](https://developer.nvidia.com/blog/advancing-the-state-of-the-art-in-automl-now-10x-faster-with-nvidia-gpus-and-rapids/) | 2021/06/09 |
+| :tv: [AutoGluon and Distillation](https://icml.cc/virtual/2020/7045)                                                                                               | Keynote  | [ICML 2020, AutoML Workshop](https://icml.cc/virtual/2020/workshop/5725)                                                           | 2020/07/18 |
```

Comment:
Looks like that a comprehensive AutoMM tutorial is lacking. Though some tutorials mention AutoMM to some degree, their introductions are inconsistent with the current AutoMM story and shape. We can add one when available.

Reply:
If any talk we have given is missing, let me know and we can add it. If we need to do one, an option is for you to record it and we can upload it to YouTube, I created an AutoGluon YouTube channel.